tests/alive-tv/vector/vp_add.srctgt.ll

```diff
@@ -0,0 +1,14 @@
+
+declare <2 x i2> @llvm.vp.add.v2i2 (<2 x i2>, <2 x i2>, <2 x i1>, i32)
+
+define <2 x i2> @src(<2 x i2> %x, <2 x i2> %y) {
```

Comment:
please add some test that fails so we know this is not just proving stuff with `assume false`  but rather doing something for real.